fixtures/browser-rendering/test/index.spec.ts

```diff
@@ -0,0 +1,57 @@
+// test/index.spec.ts
+import { rm } from "node:fs/promises";
+import { resolve } from "path";
+import { afterAll, beforeAll, describe, expect, it } from "vitest";
+import { runWranglerDev } from "../../shared/src/run-wrangler-long-lived";
+
+describe("Local Browser", () => {
+	let ip: string,
+		port: number,
+		stop: (() => Promise<unknown>) | undefined,
+		getOutput: () => string;
+
+	beforeAll(async () => {
+		// delete previous run contents because of persistence
+		await rm(resolve(__dirname, "..") + "/.wrangler", {
+			force: true,
+			recursive: true,
+		});
+		({ ip, port, stop, getOutput } = await runWranglerDev(
+			resolve(__dirname, ".."),
+			["--local", "--port=0", "--inspector-port=0"]
+		));
+	});
+
+	afterAll(async () => {
+		await stop?.();
+	});
+
+	async function fetchText(url: string) {
+		const response = await fetch(url, {
+			headers: {
+				"MF-Disable-Pretty-Error": "1",
+			},
+		});
+		const text = await response.text();
+
+		return text;
+	}
+
+	it("Doesn't run a browser, just testing that the worker is running!", async () => {
+		await expect(
+			fetchText(`http://${ip}:${port}/create?workflowName=test`)
```

Comment:
`create?workflowName=test` <- this is fine but feels like it was copied from elsewhere??